backend/test/game/test_turn.py

```diff
@@ -0,0 +1,30 @@
+import collections
+import pytest
+
+import backend.src.game.turn as turn
+from backend.src.game.dice import DieValue
+
+
+def test_calculate_victory_points_from_dice():
```

Comment:
Please break this out into separate tests. TDD best practices - one assert per test + each test should only fail for one reason

Reply:
I think it's best to normalize that the list of dice will be 6, I don't think it's possible to be less than 6. Having a special case for 7 might be beneficial because I think some cards may allow for that.

Reply:
I can break these into multiple tests if you think its best but I think we are getting a bit too into the weeds with the testing this. The list count really shouldn't matter as it is independent of the function and there is no difference if there was 6 dice vs 600 dice in a list

Reply:
I think the tests are plenty as is, they should just be separated.
It was just weird to me that some lists had 5, some 6, some 7 elements

Reply:
It seems excessive to go through all the possible combinations here, I think just having branch coverage and a few different cases works otherwise you could argue there should be 14 tests (1x sub 3, 1x super 6, 3x for 3-6 die rolls

Reply:
I agree... "I think the tests are plenty as is, they should just be separated."